src/inverseRobot.js

```diff
@@ -7,7 +7,20 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const swappedObj = {};
+  const seenValues = new Set();
```

Comment:
The 'seenValues' Set is not necessary. You can check the existence of the property directly in the 'swappedObj'.